.gitignore

```diff
@@ -43,3 +43,5 @@ _testmain.go
 *.exe
 *.test
 *.prof
+
+.vscode/
```

Comment:
Let's not have this.

Reply:
@0zAND1z you can add `.vscode/` to your `~/.gitignore` so that it is still ignored by git but you don't have to add it on a per-project basis.